SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -20,8 +20,14 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKServices);
 
 IMPLEMENT_MODULE(FSpatialGDKServicesModule, SpatialGDKServices);
 
+#if PLATFORM_WINDOWS
```

Comment:
Not specific to this PR, but would be nice to move the constants definition to anonymous namespace.